packages/components/src/__overlays__/Tooltip/v2/index.ts

```diff
@@ -0,0 +1 @@
+export * from "../v1/Tooltip"
```

Comment:
Looks like this results in deprecation warnings in VS Code for v2 imports as well, which I'm assuming wasn't intended.

Reply:
Sorry never mind, I was looking at my implementation and I had done it slightly differently